e2e/email-settings.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { test, expect } from '@playwright/test';
+
+const settingsPageElement = {
+  emailSettingsSectionHeader: 'email-settings-header',
+  emailVerificationAlert: 'email-verification-alert',
+  emailVerificationLink: 'email-verification-link',
+  currentEmailText: 'current-email',
+  newEmailInput: 'new-email-input',
+  confirmEmailInput: 'confirm-email-input',
+  saveButton: 'save-email-button',
+  emailSubscriptionYesPleaseButton: 'yes-please-button',
+  emailSubscriptionNoThanksButton: 'no-thanks-button',
+  flashMessageAlert: 'flash-message'
+} as const;
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/settings');
+});
+
```

Comment:
```suggestion
test.beforeAll(async ({ browser }) => {
  page = await browser.newPage();
  await page.goto('/');
});
```

replace this with the already being used beforEeach block